nss-day-cohort-46/holiday-road-accidental-engine
# Nah setup holiday road

# Description

I added modules for AttractionProvider.js, AttractionSelect.js Attraction.js.
There is some basic starter code for me to reference. 
I'm going to open a new branch to work on the functionality of the attraction/"Bizarreries" dropdown.

Please delete options that are not relevant.


-----------------------------------
# Testing Instructions
git fetch --all
git checkout nah-setupHolidayRoad
serve
-----------------------------------

The functionality is not complete and testing for functionality is not ready.  Please review the basic starter code and make sure I'm on the right track.. 

# Checklist:

- [N/A] I have added test instructions that prove my fix is effective or that my feature works
